mlir/utils/jenkins/Jenkinsfile.nightly-perf

```diff
@@ -0,0 +1,87 @@
+pipeline {
+    agent {
+        docker {
+            image 'rocm/mlir:rocm4.1-latest'
+            args '--user "$(id -u):$(id -g)" --device=/dev/kfd --device=/dev/dri --group-add video -u 0'
+            label 'mlir'
+            alwaysPull true
+        }
+    }
+    stages {
+        stage('Environment') {
+            steps {
+                sh 'cat /etc/os-release'
+                sh '/opt/rocm/bin/rocm-smi'
+            }
+        }
+        stage('Build MIOpen') {
+            steps {
+                checkout scm
+                sh '''
+                    cd ~
+                    git clone https://github.com/ROCmSoftwarePlatform/MIOpen
+                    cd MIOpen
+                    cmake -P install_deps.cmake --minimum
+                    mkdir -p build && cd build
+                    CXX=/opt/rocm/llvm/bin/clang++ cmake \
```

Comment:
Maybe do this with -DCMAKE_CXX_COMPLIER? Also, out of curiosity, why do we need this particular clang++? There's one installed system-wide